templates/winthrop_staging/local_settings.py

```diff
@@ -0,0 +1,119 @@
+# This file is exec'd from settings.py, so it has access to and can
+# modify all the variables in settings.py.
+# If this file is changed in development, the development server will
+# have to be manually restarted because changes will not be noticed
+# immediately.
+
+
+# Add Google Analytics
+INCLUDE_ANALYTICS = True
+
+# Make these unique, and don't share it with anybody.
+SECRET_KEY = "{{ secret_key }}"
+
+# username for accessing GeoNames API
+GEONAMES_USERNAME = '{{ geonames_username }}'
+# mapbox access token
+MAPBOX_ACCESS_TOKEN = '{{ mapbox_token }}'
+
+# Database
+# https://docs.djangoproject.com/en/1.10/ref/settings/#databases
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql',
+	'NAME': '{{ db_name }}',
+	'USER': '{{ db_username }}',
+	'PASSWORD': '{{ db_password}}',
+	'HOST': '{{ db_host }}',
+	'PORT': '3306',
+	'OPTIONS': {
+		'init_command': "SET sql_mode='STRICT_TRANS_TABLES'",
+	}
+    }
+}
+
+SOLR_CONNECTIONS = {
+    'default': {
+        'COLLECTION': 'winthrop',
+        'URL': 'http://127.0.0.1:8983/solr/',
+        'ADMIN_URL': 'http://127.0.0.1:8983/solr/admin/cores'
+    },
+   'test': {
+        'COLLECTION': 'winthrop-test',
+        'URL': 'http://127.0.0.1:8983/solr/',
+        'ADMIN_URL': 'http://127.0.0.1:8983/solr/admin/cores'
+    }
+}
+
+CAS_SERVER_URL = 'https://fed.princeton.edu/cas/'
```

Comment:
Can we also make CAS & ldap info common variables?

Reply:
I hadn't extracted them because there wasn't implicitly a reason they couldn't just appear in the template. LDAP info may be a bit of a trick because it's multiline. I will certainly try.

Reply:
Templated these out (vs a multiline variable). They can live in a .py extension file so that we don't have to worry about any jinja2 gotchas (the only thing I don't like is that I had to hard code `templates` as a path but that's likely to be 100% uniform)